byoqm/visuals/pie.py

```diff
@@ -0,0 +1,47 @@
+from math import pi
```

Comment:
I can't see we use this fie anywhere. Are we jumping the gun here? Especially, because it also is the sole reason for adding pandas as a dependency (correct me if I am wrong)?

Reply:
pie has been removed